actors/builtin/miner/policy.go

```diff
@@ -108,10 +108,13 @@ var FaultMaxAge = WPoStProvingPeriod * 14
 // key or allowing the owner account to submit PoSts while a key change is pending.
 const WorkerKeyChangeDelay = ChainFinality
 
+// Minimum number of epochs past the current epoch a sector may be set to expire.
+const MinSectorExpiration = 180 * builtin.EpochsInDay
+
 // Maximum number of epochs past the current epoch a sector may be set to expire.
 // The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
-const MaxSectorExpirationExtension = builtin.EpochsInYear
+const MaxSectorExpirationExtension = 366 * builtin.EpochsInDay
```

Comment:
why was this changed?

Reply:
This is latest cryptoecon param recommendation.  cc @zixuanzh

Reply:
The difference is that EpochsInYear is 365*builtin.EpochsInDay and the new spec says 366 days (leap-year upper bound). I don't know if this is a change motivated by real concerns or just an inconsistency in how various teams define a year, but I thought it best to stick to the spec.